azure-devops/cross-build.yml

```diff
@@ -128,3 +128,34 @@ jobs:
         testResultsFormat: JUnit
         testResultsFiles: '**/test-results.xml'
         testRunTitle: 'build-test-${{ parameters.targetPlatform }}-${{ parameters.shardNum }}'
+    - task:
+      - publish: $(buildOutputLocation)
+        artifact: ArmBuild
+      timeoutInMinutes: 120
+- job:
+  pool: Arm2
+  steps:
+    - task:
+      - download: current
+        artifact: ArmBuild
+      timeoutInMinutes: 120
+    - task: CmdLine@2
+      displayName: 'Run Tests'
+      timeoutInMinutes: 120
+      condition: succeeded()
+      inputs:
+        workingDirectory: current
+        script: |
+          call "%PROGRAMFILES(X86)%\Microsoft Visual Studio\2019\Preview\Common7\Tools\VsDevCmd.bat" ^
+          -host_arch=${{ parameters.vsDevCmdArch }} -arch=${{ parameters.vsDevCmdArch }} -no_logo
+          ctest -V

```

Comment:
While this is a good start it's not going to be as simple as just running `ctest` you're going to have to modify the python scripts under `test` to create a "run only mode" for our LIT test format. In the PR you've based this off of I made a "build only mode" so that can be a place to start looking.